test/functional/gridfs_stream_tests.js

```diff
@@ -1124,10 +1124,12 @@ describe('GridFS Stream', function() {
                 download.on('error', function(error) {
                   if (!testSpec.assert.error) {
                     test.ok(false);
+                    download.abort();
```

Comment:
is this call to `abort` necessary?

Reply:
the call to abort is necessary to close the underlying cursor.